saved-sites/saved-sites-impl/src/main/java/com/duckduckgo/savedsites/impl/newtab/FavouritesNewTabSectionViewModel.kt

```diff
@@ -70,6 +71,23 @@ class FavouritesNewTabSectionViewModel @Inject constructor(
         val bookmarkFolder: BookmarkFolder?,
     )
 
+    enum class Placement {
+        FOCUSED_STATE,
+        NEW_TAB_PAGE,
```

Comment:
🔍 Could give each enum an int value and then use that in `FavouritesNewTabSectionView` as `NEW_TAB_PAGE` instead of `1`